WebDriverAgentLib/Categories/XCElementSnapshot+FBHelpers.m

```diff
@@ -128,6 +132,26 @@ - (XCElementSnapshot *)fb_parentCellSnapshot
     }
     return targetCellSnapshot;
 }
+
+- (CGRect)fb_visibleFrameWithFallback
+{
+  CGRect thisVisibleFrame = [self visibleFrame];
+  if (!CGRectIsEmpty(thisVisibleFrame))
+    return thisVisibleFrame;
```

Comment:
Please keep curly brackets even if the if block contains one operator. Otherwise we could put both into the same line